admin_guide/pruning_resources.adoc

```diff
@@ -272,3 +274,49 @@ $ oadm prune images --keep-tag-revisions=3 --keep-younger-than=60m --confirm
 
 $ oadm prune images --prune-over-size-limit --confirm
 ----
+
+[[image-pruning-problems]]
+=== Image pruning problems
+If your images keep pilling up and prune command removes just a small portion
+of what you'd expect, make sure you understand xref:image-prune-conditions[the
+conditions] that must apply for an image to be considered a candidate for pruning.
+
+Especially, ensure that images you want removed occur at higher positions in
+each
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#image-stream-tag[tag
+history] than your chosen tag revisions threshold. For example, consider an old
+and obsolete image named `sha:abz`. By running following command in particular
+namespace `N` where the image is tagged:
```

Comment:
Running the following command in namespace N, where the image is tagged you will see the image is tagged thre times in a single istag named myapp:

... here goes the code...